File: src/js/file_system.js
const fs = require("fs");
const path = require("path");

const folder = path.join(__dirname, "./../files/");

fs.readdir(folder, (err, files) => {
    const folderItems = [];
    const fileItems = [];

    files.forEach(file => {

        const fileStat = fs.statSync(path.join(folder, file));

        if (fileStat.isDirectory())
            folderItems.push(file);

        if (fileStat.isFile())
            fileItems.push({
                "name": file,
                "size": fileStat.size
            });


    });
    folderItems.forEach(folderItem => console.log("/" + folderItem));

    fileItems.forEach(fileItem => console.log(fileItem.name + "," + fileItem.size + "B"));
});

File: src/py/file_system.py
from os import listdir
from os.path import isdir, isfile, join, dirname, getsize, abspath

folderPath = join(dirname(__file__), "..\\files")

folders = []
files = []
for childItem in listdir(folderPath):
    if isdir(join(folderPath,childItem)):
        folders.append(childItem)
    if isfile(join(folderPath,childItem)):
        files.append(childItem)

for folder in folders:
    print("/", folder, sep="")

for fileItem in files:
    print(fileItem, ",", getsize(join(folderPath,fileItem)), "B", sep="")